app/views/reviews/new/_facilities.html.erb

```diff
@@ -0,0 +1,29 @@
+<%
+  # Locals:
+  # facilities: Pre-filtered list of AggregatedFacilityReviews for space.
+  # form: Form for a Review
+%>
+
+<% FacilityCategory.all.each do |category| %>
+  <%
+    facilities_in_category = facilities.in_category(category)
```

Comment:
Burde kanskje slike som disse ligge i controlleren eller i en hjelper?

Reply:
Kan være enig i det!

Litt stuck på hvordan man kunne strukturere denne delen bedre? Her har vi jo to loops som kjører, først looper vi alle facility_categories, og så looper vi alle facilitetene i den kategorien og ser etter om vi har en review for dem. 

Siden vi skal printe noe for begge nivåer av loopen, så er jeg ikke flink nok til å se hvordan hente det ut til helpers uten at det blir mer rotete enn det allerede er, men om du ser noe så er jeg for!

Reply:
https://github.com/lnu-norge/lokaler.lnu.no/pull/80